extensions/redis-client/runtime/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-redis-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>quarkus-redis-client</artifactId>
+
+    <name>Quarkus - Redis Client - Runtime</name>
+    <description>Synchronous and Reactive Redis client</description>
```

Comment:
I think it should be something like `Connect to Redis in either imperative or reactive style` to be consistent with the other descriptions on code.quarkus.io.